frontend/src/tabs/authorship.pug

```diff
@@ -18,32 +19,32 @@
     .contribution(v-if="isLoaded && files.length!=0")
       .sorting.mui-form--inline
         .mui-select
-          select.medium-font(v-model="filesSortType")
+          select(v-model="filesSortType")
```

Comment:
this as well

Reply:
This is overwritten in the original css. Therefore, removed it.

Reply:
Hmm, when you mean overwritten by original css, it means that adding `medium-font` to this does not change the css property?

Reply:
Yes

Reply:
![Screen Shot 2020-04-10 at 4 33 20 PM](https://user-images.githubusercontent.com/35946746/78976561-fb2b4480-7b48-11ea-99f1-a17ff5d5f6a0.png)

Reply:
if you do this:
```
mui-select > select {
    @include medium-font;
}
```

will it override the original css?

Reply:
I can overwrite the original CSS. That is not a problem. But this will change the approved UI. Should I go ahead with it?

Reply:
Yes. Actually that was the intended UI, but somehow we did not realised that it was overwritten by the `mui-select` class.

Reply:
Okay. Will do accordingly.